CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ### master
 
+### 2.0.1 - 2020-03-04
+
+* update vuln gems: bootstrap-sass and nokogiri
```

Comment:
why `vuln` (which is not obvious what it means if you don't know the context) and not `vulnerable` if it only takes few more characters? I would stay away from using shortcuts as much as possible.